src/Compiler/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/RazorBaselineIntegrationTestBase.cs

```diff
@@ -162,7 +162,10 @@ protected void AssertCSharpDocumentMatchesBaseline(RazorCodeDocument codeDocumen
         actualMappings = actualMappings.Replace("\r", "").Replace("\n", "\r\n");
         Assert.Equal(baselineMappings, actualMappings);
 
-        AssertLinePragmas(codeDocument);
+        if (verifyLinePragmas)
```

Comment:
Why do we need to not verify pragmas?

Reply:
The code is invalid, so codegen doesn't map all user-typed C# code to the generated source, hence the `AssertLinePragmas` would fail. That happens for existing cases, too (they just aren't tested I guess), e.g.:

```razor
<TestComponent Test="start @("literal") @x end" />
@code {
    int x = 1;
    [Parameter] string Test { get; set; }
}
```

I do the same thing in https://github.com/dotnet/razor/pull/8927.